aws/autoscaler/main.tf

```diff
@@ -33,7 +33,9 @@ module "eks-cluster-autoscaler" {
     # Open the Cluster Autoscaler releases page from GitHub in a web browser and find the latest Cluster Autoscaler version that matches the Kubernetes major and minor version of your cluster.
     # For example, if the Kubernetes version of your cluster is 1.20, find the latest Cluster Autoscaler release that begins with 1.20.
     "image" : {
-      "tag" : "v${data.aws_eks_cluster.cluster.version}.0"
+      "tag" : "v${data.aws_eks_cluster.cluster.version}.3",
+      "repository" : "registry.k8s.io/autoscaling/cluster-autoscaler"
```

Comment:
didn't fetch the right image.

https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.27.3